### ISSUE TITLE
[Question] 'missing' build_policy 

### ISSUE THREAD

**Issue by boussaffawalid** [conan-io/conan-package-tools#438]
### 
Hello,
Does the build_policy [missing] also support building dependencies or only the current package ?
 
Here is my situation, 
I set the env var `CONAN_BUILD_POLICY` to `missing`, I was expecting conan to build missing dependencies but I got this error:  

```
17:41:58  conans.errors.ConanException: Missing prebuilt package for 'my_package/1.0.0@my_user/my_channel'
17:41:58  Try to build it from sources with "--build my_package"  
```

### Environment Details
* Conan Package Tools Version: **0.30.4**
* Operating System: **Windows**
* Operating System Version: **10**
* Compiler+version: **visual studio 2015**
* Conan version: **conan 1.21.0**
* Python version: **python 3.7.0**

**Comment by uilianries** [comment]
Yes, it should build all dependencies: https://docs.conan.io/en/latest/faq/troubleshooting.html#error-missing-prebuilt-package

Could you please share more details about your environment?
- build.py
- env vars configured
- commands executed
- entire build log
